Cartfile

```diff
@@ -0,0 +1 @@
+github "joshaber/SwiftBox" "c11e489"
```

Comment:
This is currently pointing to the tip of master. We were previously pointing to the prior commit, but the latest one seems innocuous. Might be time for a SwiftBox release just so we can point to it.